src/interface/src/app/home/planning-areas/planning-areas.component.ts

```diff
@@ -1,8 +1,176 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { MatIconModule } from '@angular/material/icon';
+import { Router, RouterLink } from '@angular/router';
+import { ButtonComponent } from '@styleguide';
+import { MatMenuModule } from '@angular/material/menu';
+import { MatButtonModule } from '@angular/material/button';
+import { MatTableModule } from '@angular/material/table';
+import { MatSortModule, Sort, SortDirection } from '@angular/material/sort';
+import {
+  AsyncPipe,
+  DatePipe,
+  DecimalPipe,
+  JsonPipe,
+  KeyValuePipe,
+  NgForOf,
+  NgIf,
+  NgSwitch,
+  NgSwitchCase,
+  NgSwitchDefault,
+} from '@angular/common';
+import { AuthService, PlanService } from '@services';
+import { PreviewPlan } from '@types';
+import { PlanningAreasDataSource } from './planning-areas.datasource';
+import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
+import {
+  DEFAULT_SORT_OPTIONS,
+  QueryParamsService,
+} from './query-params.service';
+import { KeyPipe } from '../../standalone/key.pipe';
+import {
+  MatLegacyDialog as MatDialog,
+  MatLegacyDialogModule as MatDialogModule,
```

Comment:
This is wrong, for now. We should be using the new mat dialog and mat dialog modules, but the current delete planning area modal uses the old one. 
There are new designs for that modal, so when the time comes we can create a new version that uses the non-legacy stuff.